src/components/popups/ProfileSettings.tsx

```diff
@@ -137,6 +140,7 @@ const ProfileSettings = ({ isOpen, onClose, onProfileOpen }) => {
         setRepPassword("");
         onClose();
```

Comment:
redundant on close (also one already at the end of the if)

Reply:
Deleted